robolectric/src/test/java/org/robolectric/android/internal/AndroidTestEnvironmentTest.java

```diff
@@ -112,13 +115,15 @@ public void setUpApplicationState_setsMainThread_onAnotherThread() throws Interr
    */
   @Test
   public void ensureBouncyCastleInstalled() throws GeneralSecurityException {
-    bootstrapWrapper.callSetUpApplicationState();
 
-    MessageDigest digest = MessageDigest.getInstance("SHA256");
-    assertThat(digest.getProvider().getName()).isEqualTo(BouncyCastleProvider.PROVIDER_NAME);
+      bootstrapWrapper.callSetUpApplicationState();
```

Comment:
This might just disappear once you run format, but if not, I don't think this code is actually any different with our changes so it's safe to revert this part.